### DISCUSSION TITLE
v0.0.10 🌈

### DISCUSSION THREAD

**Discussion by lalitpagaria** [obsei/obsei#207] [General]
## :star: Major Highlights
- Google Maps: Now observer able to fetch google maps reviews
- Handle Long Text: Now use TextSplitter and InferenceAggregator to seamlessly process very long text for example news article
- Analyzer: New TextClassification analyzer let you use non zero-shot classification model
- Analyzer: New Spacy powered NER analyzer let you use spacy based NER models
- Pandas: Now you can use Pandas DataFrame as an observer and informer, which enable you to load and store data from CSV, TSV, Excel and SQL DBs.
- Miscellaneous: Jinja template support for Slack message, new tutorials and pre-commit hook to save dev time

## 🚀 Features

- Jinja Template for Slack, RegEx \& Lammatizer cleaner functions and Sentence based text splitting @lalitpagaria (#206)
- Add SDK and UI-Demo image release job @lalitpagaria (#205)
- Adding jinja template support for slack messages @lalitpagaria (#199)
- Adding google maps review observer via outscrapper api @lalitpagaria (#195)
- add text classification analyzer @shahrukhx01 (#191)
- Add Pandas as Observer/Source @cnarte (#184)
- Adding app\_url support to appstore and playstore scrappers @lalitpagaria (#180)
- Adding InferenceAggregator @akar5h (#166)
- spacy ner analyzer , #165 enhancement @akar5h (#171)
- Adding tutorials information and articles @lalitpagaria (#168)
- Colab Tutorials Added @reenabapna (#167)
- TextSplitter Preprocessor Pipeline @akar5h (#160)
- Pre commit integration @salilmishra23 (#156)
- add requirement files for development @salilmishra23 (#159)
- Facebook source time range improvement @GirishPatel (#157)

## 🐛 Bug Fixes

- Reverting to older messaging format for Twitter v2 API @lalitpagaria (#198)
- Fixing import issues along @lalitpagaria (#193)
- Fix Twitter Source Config import issue @lalitpagaria (#190)
- Use of BaseSettings Causing regression so reverting the changes @lalitpagaria (#189)
- Creating quoted query before passing to GNews client @lalitpagaria (#181)
- Handle Null case when crawler failed to fetch article @lalitpagaria (#173)
- Email source fixed, no duplicates in each iteration @namanjuneja771 (#158)

## 🧰 Maintenance

- Adding support to dailyget message api @lalitpagaria (#179)

## ⚠️Breaking Changes

- Fix Twitter Source Config import issue @lalitpagaria (#190)
- Adding InferenceAggregator @akar5h (#166)
- spacy ner analyzer , #165 enhancement @akar5h (#171)

## 🙏 Release Contributors! :heart:
@akar5h @cnarte @GirishPatel @lalitpagaria @namanjuneja771 @reenabapna @salilmishra23 @shahrukhx01 

<hr /><em>This discussion was created from the release <a href='https://github.com/obsei/obsei/releases/tag/v0.0.10'>v0.0.10 🌈</a>.</em>

**Comment by reenabapna** [comment]
Good going...👍
